packages/dialect-react/components/DialectContext/index.tsx

```diff
@@ -473,7 +474,9 @@ export const DialectProvider = (props: PropsType): JSX.Element => {
 
   return (
     <DialectContext.Provider value={value}>
-      {props.children}
+      <WalletIdentityProvider>
```

Comment:
Do we need cardinal in react package, since it's only a visual thing at the moment?

I would move this provider to NotificationsButton and ChatButton where all providers are currently located. That way we localize those changes only to react-ui, since it's only necessary for UI part of dialect at the moment, but we can improve on this later. Wdyt?

Reply:
yeaa this also makes sense to me updated so that we are only inject cardinal context in the UI components